ccov_api.js

```diff
@@ -0,0 +1,406 @@
+/**
```

Comment:
This file has been added back because it is referenced in the tests.  I suspect it was moved to another dir, or split up, or changed.  I will leave it to anther pull request to deal with updating the tests, and removing this file.